src/components/Centralizer/Centralizer.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Centralizer.css";
+
+type CentralizerParam = { children: React.ReactNode };
```

Comment:
@ViniciusDaCunha  please use the plural names for the params like

CentralizerParams